scripts/rebuild-deb.sh

```diff
@@ -20,9 +20,16 @@ PROJECT="coda-$(echo "$DUNE_PROFILE" | tr _ -)"
 if [ "$GITBRANCH" == "master" ]; then
     VERSION="$GITTAG-${GITHASH}"
 else
-    VERSION="${CIRCLE_BUILD_NUM}-${GITBRANCH}-${GITHASH}-PV${PVKEYHASH}"
+    VERSION="$GITTAG-${GITBRANCH}-${GITHASH}-PV${PVKEYHASH}"
```

Comment:
Writing down what we discussed in person for reviewers:

The thinking is builds are not bit for bit identical but they are deterministic enough that we don’t need to store old builds for same git sha (we capture pvkey hash anyway in case the s3key stuff ever regresses)

Reply:
disagree about removing CIRCLE_BUILD_NUM

Reply:
@bkase @jkrauska I've been thinking about this a lot and feel like I've struck upon a nice balance, *especially* now that we have reasonably deterministic builds. 

As it is, I could just re-run a build for an old commit, and that version would be "newer" than the most recent code, causing the update functionality to not work as expected. 

The use-case for a pre-release version is the following: 
- Make commit to develop
- Produce `.deb`
- Manually deploy that version against running testnet for `X hours`
- Tear down once testing is complete and compare metrics 

The only use-case for `apt-get update` is in a public release against the `release` repo, and that works as-designed right now.

Reply:
I use `apt-get update` against `develop` all the time on linux vms and hardware.

I'd like to keep that working.

Reply:
it also allow the TF pattern:

```
  coda_repo    = "develop"
  coda_version = "*"  # Note: '*' gets latest when specifying version
```

which is used for testnet and other testing against `develop`